modules/server/src/main/scala/navigate/server/tcs/M2GuideConfig.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.server.tcs
+
+import cats.{Show, Eq}
+import cats.syntax.all.*
+import navigate.model.enums.TipTiltSource
+
+/** Data type for M2 guide config. */
+sealed trait M2GuideConfig extends Product with Serializable {
+  def uses(s: TipTiltSource): Boolean
+}
+
+object M2GuideConfig {
+  case object M2GuideOff                                                    extends M2GuideConfig {
```

Comment:
Odd alignment?

Reply:
Weird. I did run scalafmt on all the files.